src/resources/elements/primeDesignSystem/pselect/pselect.ts

```diff
@@ -0,0 +1,46 @@
+import { bindable, customElement } from "aurelia-framework";
+import SlimSelect from "slim-select";
+import "./pselect.scss";
+
+/**
+ * Usage:
+ *    <pselect type="primary">Primary</pselect>
+ *    <pselect type="secondary">Secondary</pselect>
+ *    <pselect type="tertiary">Tertiary</pselect>
+ *    <pselect type="primary" disabled>Primary - Disabled</pselect>
+ *    <pselect type="secondary" disabled>Secondary - Disabled</pselect>
+ *    <pselect type="tertiary" disabled>Tertiary - Disabled</pselect>
+ *    <pselect type="primary" click.delegate="message('Hi!')">Clickable</pselect>
+ *    <pselect ... full-width>Full-Width</pselect>
+*/
+
+@customElement("pselect")
+export class PButton {
+  // @bindable.booleanAttr disabled = false;
+  // @bindable.booleanAttr fullWidth = false;
+  @bindable data: Array<{value: string, text: string, innerHTML: string}>;
```

Comment:
Should export this as an interface like IpSelectItemConfig so consumer can get intellisense/compile errors.